projects/packages/publicize/src/class-publicize-setup.php

```diff
@@ -44,6 +46,10 @@ public static function on_jetpack_feature_publicize_enabled() {
 		add_action( 'rest_api_init', array( new REST_Controller(), 'register_rest_routes' ) );
 
 		add_action( 'current_screen', array( static::class, 'init_sharing_limits' ) );
+
+		add_action( 'admin_init', array( static::class, 'init_social_image_generator' ) );
```

Comment:
Hooking into the `admin_init` action breaks the `wp_after_insert_post` hook—it no longer runs for the actual post, only for the auto draft.